sphinx/templates/quickstart/make.bat.new_t

```diff
@@ -4,14 +4,115 @@ pushd %~dp0
 
 REM Command file for Sphinx documentation
 
+REM Parsing of multiple commandline options and targets
+
+REM If no arguments passed at all, skip entirely
+if "%1"=="" goto make_main
+
+setlocal
+
+set TARGETS=
+set OPTIONS=
+
+:arg_loop
+if "%1"=="" goto arg_continue
+
+set WORK=%1
+set TAKESARG=0
+set KEYVALUE=0
+
+REM Have to have these outside the main IF block, else they don't store correctly
+if "%WORK:~,1%"=="-" (
+   if "%WORK:~-1%"=="b" (
+      echo ERROR: Do not use '-b' option to specify make targets. Exiting...
+      endlocal
+      goto end_error
+      )
+   rem This list needs to be updated if new options are implemented
+   rem  that take values
+   if "%WORK:~-1%"=="d" set TAKESARG=1
+   if "%WORK:~-1%"=="j" set TAKESARG=1
+   if "%WORK:~-1%"=="c" set TAKESARG=1
+   if "%WORK:~-1%"=="D" set TAKESARG=2
+   if "%WORK:~-1%"=="t" set TAKESARG=1
+   if "%WORK:~-1%"=="A" set TAKESARG=2
+   if "%WORK:~-1%"=="w" set TAKESARG=1
```

Comment:
Are these options for sphinx-build command?
If so, make.bat knows too much for its responsibility.